CMakeLists.txt

```diff
@@ -0,0 +1,105 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(Nuklear VERSION 0.0.1 LANGUAGES C)
```

Comment:
Since `VERSION` won't match, possibly remove it?

Reply:
This could be made the primary place for the version?
Then it generates a header with the version in it, so that compile-time you have access to the same version.